api/database/migrations/2023_07_19_140947_create_digital_contracting_questionnaires_table.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Query\Expression;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // main table for digital contracting questionnaires
+        Schema::create('digital_contracting_questionnaires', function (Blueprint $table) {
+            $table->uuid('id')->primary('id')->default(new Expression('gen_random_uuid()'));
+            $table->timestamps();
+            $table->boolean('read_preamble')->nullable();
+            $table->uuid('department_id')->nullable();
+            $table->foreign('department_id')->references("id")->on("departments");
+            $table->string('department_other')->nullable();
+            $table->string('branch_other')->nullable();
+            $table->string('business_owner_name')->nullable();
+            $table->string('business_owner_job_title')->nullable();
+            $table->string('business_owner_email')->nullable();
+            $table->string('financial_authority_name')->nullable();
+            $table->string('financial_authority_job_title')->nullable();
+            $table->string('financial_authority_email')->nullable();
+            $table->jsonb('authorities_involved')->default(new Expression('\'[]\'::jsonb'));
+            $table->string('authority_involved_other')->nullable();
+            $table->string('contract_behalf_of_gc')->nullable();
+            $table->string('contract_service_of_gc')->nullable();
+            $table->string('contract_for_digital_initiative')->nullable();
+            $table->string('digital_initiative_name')->nullable();
+            $table->string('digital_initiative_plan_submitted')->nullable();
+            $table->string('digital_initiative_plan_updated')->nullable();
+            $table->string('digital_initiative_plan_complemented')->nullable();
+            $table->string('contract_title')->nullable();
+            $table->date('contract_start_date')->nullable();
+            $table->date('contract_end_date')->nullable();
+            $table->string('contract_extendable')->nullable();
+            $table->string('contract_amendable')->nullable();
+            $table->string('contract_multiyear')->nullable();
```

Comment:
Was there a reason these were made strings instead of booleans since we only accept a "yes" or "no"?